.github/ISSUE_TEMPLATE/release-check-list.md

```diff
@@ -65,7 +65,7 @@ assignees: ''
           * peer-pods: https://github.com/confidential-containers/operator/blob/main/config/samples/ccruntime/peer-pods/kustomization.yaml
           Note that we need the quay.io/confidential-containers/runtime-payload-ci registry and kata-containers-latest tag
 
-- [ ] - 8. Contact @stevenhorsman and/or @bpradipt to update peer pods to use the latest commits of kata-containers and attestation-agent and test it 
+- [ ] - 8. Update peer-pods with latest commits of kata-containers and attestation-agent and test it, following the [release candidate testing process](https://github.com/confidential-containers/cloud-api-adaptor/blob/main/docs/Release-Process.md#release-candidate-testing)
```

Comment:
Reading this line I realized that we should change the references to attestation-agent on cloud-api-adaptor due to the recent merge of attestation-agent, image-rs, etc, repositories. Opened https://github.com/confidential-containers/cloud-api-adaptor/issues/1126